src/NServiceBus.Persistence.DynamoDB.Tests/InstallerTests.cs

```diff
@@ -0,0 +1,155 @@
+        Installer installer;
+        OutboxPersistenceConfiguration outboxSettings;
+
+        [SetUp]
+        public void Setup()
+        {
+            dynamoClient = new AmazonDynamoDBClient();
+            installerSettings = new InstallerSettings
+            {
+                CreateSagaTable = false,
+                CreateOutboxTable = true,
+            };
+            outboxSettings = new OutboxPersistenceConfiguration()
+            {
+                TableName = Guid.NewGuid().ToString("N"),
+                PartitionKeyName = Guid.NewGuid().ToString("N"),
+                SortKeyName = Guid.NewGuid().ToString("N")
+            };
+            installer =
+                new Installer(
+                    new DynamoDBClientProvidedByConfiguration
+                    {
+                        Client = dynamoClient
+                    }, installerSettings, outboxSettings);
+        }
+
+        [TearDown]
+        public async Task Teardown()
+        {
+            try
+            {
+                await dynamoClient.DeleteTableAsync(outboxSettings.TableName);
+            }
+            catch (Exception e)
+            {
+                TestContext.WriteLine($"Error deleting queue: {e}");
+            }
+        }
+
+        [Test]
+        public async Task Should_create_outbox_with_configured_attribute_names()
+        {
+            outboxSettings.PartitionKeyName = Guid.NewGuid().ToString("N");
+            outboxSettings.SortKeyName = Guid.NewGuid().ToString("N");
+
+            await installer.Install();
+
+            var table = await dynamoClient.DescribeTableAsync(outboxSettings.TableName);
+
+            Assert.AreEqual(outboxSettings.TableName, table.Table.TableName);
+            Assert.AreEqual(outboxSettings.PartitionKeyName, table.Table.KeySchema[0].AttributeName);
+            Assert.AreEqual(KeyType.HASH, table.Table.KeySchema[0].KeyType);
+            Assert.AreEqual(outboxSettings.SortKeyName, table.Table.KeySchema[1].AttributeName);
+            Assert.AreEqual(KeyType.RANGE, table.Table.KeySchema[1].KeyType);
+            //TODO do we also need to test the attribute type to be "S"?
+            Assert.AreEqual(TableStatus.ACTIVE, table.Table.TableStatus);
+        }
+
+        [Test]
+        public async Task Should_create_outbox_with_pay_as_you_go_billing_mode()
+        {
+            installerSettings.BillingMode = BillingMode.PAY_PER_REQUEST;
+
+            await installer.Install();
+
+            var table = await dynamoClient.DescribeTableAsync(outboxSettings.TableName);
+
+            Assert.AreEqual(BillingMode.PAY_PER_REQUEST, table.Table.BillingModeSummary.BillingMode);
+            Assert.AreEqual(0, table.Table.ProvisionedThroughput.ReadCapacityUnits);
+            Assert.AreEqual(0, table.Table.ProvisionedThroughput.WriteCapacityUnits);
+        }
+
+        [Test]
+        //TODO this doesn't work with the local DB because BillingModeSummary will be null
```

Comment:
summary MIGHT be null when using provisioned mode, we can't assert for it. focus on the capacity units